client/asset/interface.go

```diff
@@ -40,6 +46,9 @@ func DetermineWalletTraits(w Wallet) (t WalletTrait) {
 	if _, is := w.(NewAddresser); is {
 		t |= WalletTraitNewAddresser
 	}
+	if _, is := w.(LogFiler); is {
+		t |= WalletTraitLogFiler
```

Comment:
Glad we're getting use out of these traits.  I had my reservations about a bitset limited to 64 traits though, crossing fingers we have enough.  :laughing: